mix.exs

```diff
@@ -57,7 +57,7 @@ defmodule Protobuf.Mixfile do
       maintainers: ["Tony Han"],
       licenses: ["MIT"],
       links: %{"GitHub" => "https://github.com/tony612/protobuf-elixir"},
-      files: ~w(mix.exs README.md lib config LICENSE priv/templates .formatter.exs)
+      files: ~w(mix.exs README.md lib/protobuf.ex lib/protobuf config LICENSE priv/templates .formatter.exs)
```

Comment:
@tony612 protos in `./lib/google` is only used by protoc plugin, so I think we don't need to package them in.

Reply:
That's exactly what I want to do recently :)